lang/levels/classic/text_fre.po

```diff
@@ -0,0 +1,537 @@
+msgstr "Maintenant que vous avez massacré les troupes du syndicat, son directeur semble assez en colère. Atrocement joué Gardien ! Vous devez maintenant décimer son armée, une tâche nécessaire pour prendre le contrôle de ces terres."
+
+#. Syndika (lv206)
+#: guitext:4
+msgctxt "In-game message"
+msgid "The syndicate has been eradicated! Congratulations Keeper on your victory against the Director. "
+msgstr "Le syndicat a été éradiqué ! Félicitations Gardien pour votre victoire contre le directeur. "
+
+#. Kzelbatu (lv207)
+#: guitext:5
+msgctxt "In-game message"
+msgid "Magic powers in this realm have reduced your abilities. The meager resources of this realm are fiercely guarded by smugglers whom you must overcome if you are to be victorious Keeper! Maybe separate them from their heads?"
+msgstr "Les pouvoirs magiques de ce royaume ont réduit vos capacités. Les maigres ressources de ce royaume sont férocement gardées par des contrebandiers que vous devez vaincre si vous voulez être le Gardien victorieux ! Peut-être séparer leur tête de leurs corps ?"
+
+#. Kzelbatu (lv207)
+#: guitext:6
+msgctxt "In-game message"
+msgid "The smugglers have sent a raiding party towards you. You know what must be done! "
+msgstr "Les contrebandiers ont envoyé une équipe de raid vers vous. Vous savez ce qu'il faut faire ! "
+
+#. Kzelbatu (lv207)
+#: guitext:7
+msgctxt "In-game message"
+msgid "The elite troupe has been resurrected. Destroy them again, and this time let it be once and for all! "
+msgstr "La troupe d'élite a été ressuscitée. Détruisez-les à nouveau, et cette fois, que ce soit pour de bon ! "
+
+#. Kzelbatu (lv207)
+#: guitext:8
+msgctxt "In-game message"
+msgid "A second bandit king has been raised in the secret altar, with a formidable army. When he counts his troops after the forthcoming battles, teach him to understand the number zero!"
+msgstr "Un deuxième roi bandit a été créé dans l'autel secret, avec une armée redoutable. Lorsqu'il compte ses troupes après les batailles à venir, apprenez-lui à comprendre le chiffre zéro !"
+
+#. Kzelbatu (lv207)
+#: guitext:9
+msgctxt "In-game message"
+msgid "Heroic reinforcements have arrived on your land to take your belongings. Reward them for their impudence! "
+msgstr "Des renforts héroïques sont arrivés sur vos terres pour prendre vos affaires. Récompensez-les pour leur impudence ! "
+
+#. Kzelbatu (lv207)
+#: guitext:10
+msgctxt "In-game message"
+msgid "Agents have infiltrated your dungeon to steal gold. Find a way to make their suffering special!"
+msgstr "Des agents se sont infiltrés dans votre donjon pour voler de l'or. Trouvez un moyen de rendre leur souffrance spéciale !"
+
+#. Kzelbatu (lv207)
+#: guitext:11
+msgctxt "In-game message"
+msgid "Hell must have a pact with these smugglers?  Make preparations Keeper, as these Demons have come to obliterate you! "
+msgstr "L'enfer doit avoir un pacte avec ces contebandiers ? Prépare-toi Gardien, car ces Démons sont venus pour t'anéantir ! "
```

Comment:
Ici "Hell" est plutôt une référence au diable plutôt qu'à l'enfer. L'expression anglaise "Who the hell?" se traduit par "Qui diable ?".  Je propose donc le changement suivant : "Le diable doit avoir un pacte avec ces contrebandiers ?" ou "Ces contrebandiers doivent avoir un pacte avec le diable ?". 
Quoi qu'il en soit il y a une faute de frappe : "contebandiers" -> contrebandiers.